docs/dev/version-migration/v0.1-to-v0.2-migration.md

```diff
@@ -0,0 +1,168 @@
+# v0.2.* (latest v0.2.7)
+
+* New features
+    * Adds support for `DISTINCT`, `LET` (from `FROM` clause), system stored procedure calls (`EXEC`)
+    * Adds parser support for DML statements (`INSERT`, `UPDATE`, `DELETE`), `Parameter`, `BY` variable
+    * Improvements to `LIKE` pattern compilation performance
+    * Adds function to convert from UNIX epoch to TIMESTAMP and TIMESTAMP to UNIX epoch
+* Deprecated items
```

Comment:
Also deprecated was the `V0AstSerializer`, `AstSerializer`, `AstDeserializer` -- will update w/ code examples